### PR TITLE
Add route for consolidation

### PR DESCRIPTION
**Pull request by antalakas**
[ch3306]

### CODE REVIEW

File: openapi-v1.yaml [hunk 1]
```diff
@@ -882,6 +882,26 @@ definitions:
...
+  ConsolidationConfigEntry:
+    type: object
+    description: key/value pair representing a consolidation config entry
+    properties:
+      key:
+        type: string
+      value:
+        type: string
+
+  ConsolidationConfig:
```
Comment (Shelnutt2) [inline openapi-v1.yaml:894]: Should we use just a map instead of a list of key/values?

Comment (antalakas) [reply openapi-v1.yaml:885]: Yeah works well

Comment (Shelnutt2) [reply openapi-v1.yaml:885]: Maybe rename this to just `Config` or `TileDBConfig` we might want to use this other places?

Comment (antalakas) [reply openapi-v1.yaml]: 👍

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**